### ISSUE TITLE
TToo many computers 

### ISSUE THREAD

**Issue by lxh26** [kingparks/cursor-vip#69]
Too many computers used within the last 24 hours for the same Cursor account. Please try again later, use a different Cursor account, or contact us at hi@cursor.sh if you believe this is a mistake. We have this limit in place to prevent abuse, but if you have a legitimate use case for using many computers, please do email us and we will unblock you.
Request ID: 774574df-31b1-4460-9f4f-3f18b63adfc7

**Comment by sonemeng** [comment]
SM2 will be better in this situation, but there may be some problems for composer. Only use the chat mode. The SM2 proxy mode is recommended.

**Comment by ruwiss** [comment]
I have proxy mode turned on but I get the error “Too many computers used within the last 24 hours for the same Cursor account.” in Composer. Chat works but it's not enough :(

**Comment by kingparks** [comment]
> I have proxy mode turned on but I get the error “Too many computers used within the last 24 hours for the same Cursor account.” in Composer. Chat works but it's not enough :(

Yes, composer agent is same like vip's normal model, need wait vip to show red txt and restart cursor-vip can fix it.

**Comment by ruwiss** [comment]
What do you mean by red txt

**Comment by kingparks** [comment]
> What do you mean by red txt

When it fixed, “Encountered a problem? Restart cursor-vip to solve the problem” will show in cursor-vip
